realm/realm-library/src/main/cpp/jni_util/exception_utils.cpp

```diff
@@ -0,0 +1,65 @@
+#include <unwind.h>
```

Comment:
See https://docs.fabric.io/android/crashlytics/ndk.html#one-line-stack-traces-for-native-crashes for API < 16.